src/country_index.c

```diff
@@ -49,15 +49,19 @@
 #include "md_array.h"
 #include "hashtbl.h"
 
+extern int debug_flag;
 static hashfunc country_hashfunc;
 static hashkeycmp country_cmpfunc;
 
 #define MAX_ARRAY_SZ 65536
 static hashtbl *theHash = NULL;
 static int next_idx = 0;
 static GeoIP *geoip;
+static GeoIP *geoip6;
 static char *ipstr;
-static char unknown[20] = "__";
+static char *unknown = "??";
```

Comment:
I think it is interesting to know. And, as config.h was not included yet, I think nobody used this feature except us. But if you like backwardscompatibility I do not mind changing it back to the old version.